tests/kuttl/tests/deployments/deploy1/kustomization.yaml

```diff
@@ -0,0 +1,2 @@
+resources:
+- ironic_v1beta1_ironic.yaml
```

Comment:
With this config, kustomization does not seem necessary at all, if you modify the secret in the sample file, then we can avoid kustomization all together and simply apply the sample.

Reply:
Again, I was getting that error, how do I apply the sample?  I will dig on my own as well :-)

Reply:
is it just `oc apply....`

Reply:
There is also an apply command in kuttl that takes a list of files to apply, it would be a great fit for this case: https://kuttl.dev/docs/testing/reference.html#teststep